source/_integrations/unifiprotect.markdown

```diff
@@ -256,59 +256,29 @@ Use to remove a privacy zone from a camera.
 | `device_id`            | No       | Camera you want to remove privacy zone from.                                                            |
 | `name`                 | No       | The name of the zone to remove.                                                                         |
 
-### Action unifiprotect.get_user_keyring_info
```

Comment:
It looks like `next` was not up to date when this PR was made as this PR is removing previously added docs

Reply:
fixed. sorry about that